src/rocksdb.rs

```diff
@@ -1000,6 +1024,20 @@ impl Writable for DB {
     fn single_delete_cf(&self, cf: &CFHandle, key: &[u8]) -> Result<(), String> {
         self.single_delete_cf_opt(cf, key, &WriteOptions::new())
     }
+
+    fn delete_range(&self, begin_key: &[u8], end_key: &[u8]) -> Result<(), String> {
+        let batch = WriteBatch::new();
```

Comment:
IMO, I prefer calling `delete_range_cf` with the "default" CF directly, but I also think using WriteBatch is fine here.